src/services/storage/utils.ts

```diff
@@ -38,3 +38,25 @@ export function getAvgMinMaxBillingPriceQuery (minMax: MinMax): string {
         LIMIT 1
      `
 }
+
+/**
+ * Get query for generating minimum or maximum available capacity
+ * @param minMax
+ */
+export function getMinMaxAvailableCapacityQuery (minMax: MinMax): string {
+  return `
+  SELECT
+      CAST(
+        totalCapacity - SUM(
+          COALESCE("storage_agreement"."size", 0)
+          ) as STRING
```

Comment:
Can you please remove casting to String here, and instead wrap availableCapacity field